components/card.css

```diff
@@ -0,0 +1,82 @@
+.cardcontainer {
+  display: flex;
+  align-items: center;
+  flex-direction: column;
+}
+.card {
+  display: flex;
+  align-items: center;
+  flex-direction: column;
+  justify-content: center;
+  width: 90ve;
+  height: 400px;
+  margin: 30px;
+  background-color: var(--card-color);
+  box-shadow: 0 0 20px black;
+  border-radius: 20px;
+  position: relative;
+}
+
+.card__question {
+  text-align: center;
+  margin-bottom: 20px;
+}
+.card__showanswer {
+  height: 50px;
+  width: 200px;
+  margin-top: 20px;
+  margin-bottom: 30px;
+  background-color: var(--card-show-answer-color);
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  border: 1px solid black;
+  border-radius: 10px;
+}
+.card__answer {
+  text-align: center;
+}
+
+.card__showanswer:hover {
+  background-color: var(--hover-color);
+  cursor: pointer;
+}
+.card__bookmark {
+  position: absolute;
+  height: 60px;
+  width: 60px;
+  right: 20px;
+  top: -20px;
+}
+.card__bookmark:hover {
+  cursor: pointer;
+  content: url(../images/bookmark_black.svg);
+}
+
+.card__tags {
+  position: absolute;
+  bottom: 20px;
+  left: 20px;
+  display: flex;
+  font-size: 20px;
+}
```

Comment:
If you use a list for your tag list and flexbox, you will not need to position it with `absolute`.